src/utils/do-request.js

```diff
@@ -39,28 +39,32 @@ module.exports = async function doRequest(packageName, type) {
       throw parseFailedResponse();
     }
 
-    const bestMatchUrl = xpathHelper(json, config.xpaths);
-    let url = repositoryUrl(bestMatchUrl);
+    const urls = xpathHelper(json, config.xpaths);
+    for (const bestMatchUrl of urls) {
+      try {
+        let url = repositoryUrl(bestMatchUrl);
 
-    if (!url && isUrl(bestMatchUrl)) {
-      url = bestMatchUrl;
-    }
+        if (!url && isUrl(bestMatchUrl)) {
+          url = bestMatchUrl;
+        }
 
-    if (!url && config.fallback) {
-      url = util.format(config.fallback, packageName);
-    }
+        if (!url) {
+          throw repositoryUrlNotFoundResponse();
+        }
 
-    if (!url) {
-      throw repositoryUrlNotFoundResponse();
+        // Normally, you wouldn't use `await` inside of a loop.
+        // However, we explicity want to do this sequentially.
+        // See http://eslint.org/docs/rules/no-await-in-loop
+        await got.get(url); // eslint-disable-line no-await-in-loop
+        return url;
+      } catch (err) {
+        // There's nothing to do here, so just keep going.
+        // eslint-disable-line no-empty
+      }
     }
 
-    try {
-      await got.get(url);
-      return url;
-    } catch (err) {
-      url = util.format(config.fallback, packageName);
-      return url;
-    }
+    // If we get here, no urls could be loaded.
+    return util.format(config.fallback, packageName);
```

Comment:
I like the idea of using `nock`, but I wasn't able to get it to work (see https://github.com/OctoLinker/live-resolver/pull/34/commits/ee9dfd3e6969c605efec0ee8effa6c5c0c0f0c96). It looks like mocking `find-reachable-urls` is rather difficult, so how about we just remove the failing tests, and add a functional test to ensure the relevant behavior is still covered? See https://github.com/OctoLinker/live-resolver/pull/34/commits/457bb62a2d78329ee23114f32e200e40a477978b